libkbfs/data_types.go

```diff
@@ -62,6 +62,15 @@ type TeamInfo struct {
 	// whether a user was a member given some Merkle seqno?
 }
 
+// ImplicitTeamInfo contains information needed after
+// resolving/identifying an implicit team.  TeamInfo is used for
+// anything else.
+type ImplicitTeamInfo struct {
+	Name  libkb.NormalizedUsername
```

Comment:
This is the team display name right? Perhaps worth a comment since the type says username. Same for TeamInfo.